### ISSUE TITLE
[/collect/person/] Nicola Fox bio submission

### ISSUE THREAD

**Issue by ntbot** [newtheatre/history-project#346]
End user submitted issue from page: [/collect/person/](http://alpha.history.newtheatre.org.uk/collect/person/)
---

# 'Submit an almni bio' form submission

Field | Data
----- | ----
Name | Nicola Fox
Grad Year | 2014
Course | BA (Hons) English

## Bio1 (Time at theatre)

A bit of everything really! (Except sound and lighting- I think I was always a bit too much of a technophobe for that!). Acting, stage managing, producing, costume and props, make up, front of house...etc
The theatre made my University experience more special than I ever could have imagined. I was interested in it after an Open Day tour, but I never could have predicted that I would step inside in one of my first weeks and (happily) almost never leave. I have met friends for life and had some amazing experiences thanks to the theatre and despite some very late nights, resulting tight coursework deadlines/missed lectures and abandoning my housemates for weeks on end, I wouldn't have changed a single second of it.

## Bio2 (Post-graduation)

Past- Front of House Assistant- Zoo Venues (Edinburgh Fringe Festival 2013)
Past- Front of House Assistant- The Pleasance Theatre Trust (Edinburgh Fringe Festival 2014)
Current- Legal Assistant at Freeths LLP

## Shows

Stage Manager- The Hothouse (2011)
'Kate'- The Gut Girls (2012)
Stage Manager- Closer (2012) (Nottingham and Oxford)
'Celia'- Moon (2012)
Stage Manager- 'Night, Mother (2012)
'Georgia'- Little Red (2013)
Choreographer- Frankenstein (2013)
Producer- Foxfinder (2013)
Costume and Make Up Designer- Molly Sweeney (2013)
'Elodie'- Customs (2014)

## Committees

Front of House Manager- 2012-2013
Costume, Props and Make-Up Manager- 2013-2014

## Awards

Commendation- 2014

## Attempted File Generation

```
---
title: Nicola Fox
course:
  - BA (Hons) English
graduated: 2014
---

A bit of everything really! (Except sound and lighting- I think I was always a bit too much of a technophobe for that!). Acting, stage managing, producing, costume and props, make up, front of house...etc
The theatre made my University experience more special than I ever could have imagined. I was interested in it after an Open Day tour, but I never could have predicted that I would step inside in one of my first weeks and (happily) almost never leave. I have met friends for life and had some amazing experiences thanks to the theatre and despite some very late nights, resulting tight coursework deadlines/missed lectures and abandoning my housemates for weeks on end, I wouldn't have changed a single second of it.

Past- Front of House Assistant- Zoo Venues (Edinburgh Fringe Festival 2013)
Past- Front of House Assistant- The Pleasance Theatre Trust (Edinburgh Fringe Festival 2014)
Current- Legal Assistant at Freeths LLP



Commendation- 2014
```

**

**Comment by nhyty** [comment]
Added show info

**Comment by nhyty** [comment]
closed in c391d90
